content/en/docs/tasks/manage-gpus/scheduling-gpus.md

```diff
@@ -10,133 +10,59 @@ description: Configure and schedule GPUs for use as a resource by nodes in a clu
 
 {{< feature-state state="beta" for_k8s_version="v1.10" >}}
 
-Kubernetes includes **experimental** support for managing AMD and NVIDIA GPUs
+Kubernetes includes **experimental** support for managing GPUs
 (graphical processing units) across several nodes.
 
-This page describes how users can consume GPUs across different Kubernetes versions
-and the current limitations.
-
-
-
+This page describes how users can consume GPUs, and outlines
+some of the limitations in the implementation.
 
 <!-- body -->
 
 ## Using device plugins
 
-Kubernetes implements {{< glossary_tooltip text="Device Plugins" term_id="device-plugin" >}}
+Kubernetes implements {{< glossary_tooltip text="device plugins" term_id="device-plugin" >}}
 to let Pods access specialized hardware features such as GPUs.
 
+{{% thirdparty-content %}}
+
 As an administrator, you have to install GPU drivers from the corresponding
 hardware vendor on the nodes and run the corresponding device plugin from the
-GPU vendor:
+GPU vendor. Here are some links to vendors' instructions:
 
-* [AMD](#deploying-amd-gpu-device-plugin)
-* [NVIDIA](#deploying-nvidia-gpu-device-plugin)
+* [AMD](https://github.com/RadeonOpenCompute/k8s-device-plugin#deployment)
+* [NVIDIA](https://github.com/NVIDIA/k8s-device-plugin#quick-start)
 
-When the above conditions are true, Kubernetes will expose `amd.com/gpu` or
-`nvidia.com/gpu` as a schedulable resource.
+Once you have installed the plugin, your cluster exposes a custom schedulable resource such as `amd.com/gpu` or `nvidia.com/gpu`.
 
 You can consume these GPUs from your containers by requesting
-`<vendor>.com/gpu` the same way you request `cpu` or `memory`.
-However, there are some limitations in how you specify the resource requirements
-when using GPUs:
-
-- GPUs are only supposed to be specified in the `limits` section, which means:
-  * You can specify GPU `limits` without specifying `requests` because
-    Kubernetes will use the limit as the request value by default.
-  * You can specify GPU in both `limits` and `requests` but these two values
-    must be equal.
-  * You cannot specify GPU `requests` without specifying `limits`.
-- Containers (and Pods) do not share GPUs. There's no overcommitting of GPUs.
-- Each container can request one or more GPUs. It is not possible to request a
-  fraction of a GPU.
-
-Here's an example:
+the custom GPU resource, the same way you request `cpu` or `memory`.
+However, there are some limitations in how you specify the resource
+requirements for custom devices.
+
+GPUs are only supposed to be specified in the `limits` section, which means:
```

Comment:
```suggestion
Specify GPUs within the `limits` section, and observe the following limitations:
```
I'm suggesting this to avoid the expression, "supposed to".